libraries/namespaces.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 Library
+annotationTypes:
+  namespace:
+    description: |
+      this annotation type allows you to associate unique identifier with a RAML document.
+    type: string
+(namespace): http://raml.org/vocabularies/annotations/core
```

Comment:
There is the thing in question here actually. If we do not want to use string should we enforce uri's or uri prefixes. I personally would prefer to enforce uri's.  

But this actually enforces some naming scheme that might be not optimal for some approaches.
For example if Exhange guys would generate this annotations for their assets, they might probably want to have namespace value looking as java package name.

So I decided to leave it as string for now, and then when we have a decition, we may issue version 2 of this library.